BusBoard.Web/ViewModels/BusInfo.cs

```diff
@@ -1,13 +1,46 @@
-﻿namespace BusBoard.Web.ViewModels
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using BusBoard.Api.Postcode;
+using BusBoard.Api.Tfl;
+using BusBoard.Api.Tfl.Responses;
+
+namespace BusBoard.Web.ViewModels
 {
   public class BusInfo
   {
+    public BusStopInfo BusStop1 { get; }
+    public BusStopInfo BusStop2 { get; }
```

Comment:
Minor but I'd probably name them `firstBusStop` and `secondBusStop`, or even `closestBusStop` and `secondClosestBusStop` (if that is correct that they'll be ordered like that?) to avoid appending numbers to the end of the names 🙂

Reply:
Yeah that's more clear. Changed in 04fc291 🙂